app/packages/plugins/src/index.ts

```diff
@@ -297,6 +297,37 @@ export enum PluginComponentType {
    */
 }
 
+type CategoryID = "import" | "curate" | "analyze" | "custom";
```

Comment:
Given that @brimoor this was your original vision for the categories of workflows with Assistant, do we want to simply just add all of these at right now?

Not sure if this list is still the "right" list in terms of categorization, but food for thought @ritch 

<img width="303" alt="Screenshot 2024-11-04 at 3 13 35 PM" src="https://github.com/user-attachments/assets/f559bd07-6296-4480-8b08-37946ba7f6e1">

Reply:
We've been discussing the list in a slack thread